src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: '',
+};
+
+const categories = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    CHECK_STATUS: (state, action) => ({
+      ...state,
+      categories: action.payload,
+    }),
+  },
+});
+
+export const { CHECK_STATUS } = categories.actions;
+export default categories;
```

Comment:
- [ ] Good job so far to implement some of the requirements requested by the first reviewer but some elements are missing to make your implementation complete :cry: Don't forget that you need to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Please fix it :smile: . See the perfect example below :arrow_down: 
```
// widgets.js

// Actions
const LOAD   = 'my-app/widgets/LOAD';
const CREATE = 'my-app/widgets/CREATE';
const UPDATE = 'my-app/widgets/UPDATE';
const REMOVE = 'my-app/widgets/REMOVE';

// Reducer
export default function reducer(state = {}, action = {}) {
  switch (action.type) {
    // do reducer stuff
    default: return state;
  }
}

// Action Creators
export function loadWidgets() {
  return { type: LOAD };
}

export function createWidget(widget) {
  return { type: CREATE, widget };
}

export function updateWidget(widget) {
  return { type: UPDATE, widget };
}

export function removeWidget(widget) {
  return { type: REMOVE, widget };
}

// side effects, only as applicable
// e.g. thunks, epics, etc
export function getWidget () {
  return dispatch => get('/widget').then(widget => dispatch(updateWidget(widget)))
}
```